lib/quark/compose.ex

```diff
@@ -73,7 +73,7 @@ defmodule Quark.Compose do
   def g <|> f, do: compose(g, f)
 
   @doc ~S"""
-  Function composition, from the back of the lift to the front
+  Function composition, from the back of the list to the front
```

Comment:
Except there is no list, they just take the arguments straight.  That is what is confusing to me.  :-)
It was probably a mis-copy/paste or so originally?  All it does is turn:
```elixir
sum_plus_one = compose_forward(&(Enum.sum(&1)), &(&1 + 1))
[1,2,3] |> sum_plus_one.()
```
To basically just become this:
```elixir
[1,2,3]
|> (&(Enum.sum(&1)).()
|> (&(&1 + 1)).()
```
The list is only used in the example, it is not part of the definition and does not belong in the definition's description.